scripts/newsexsystem.gd

```diff
@@ -2226,7 +2261,17 @@ func endencounter():
 				rewardtext += "\nIngredient Gained from [color=aqua]"+i.name+"[/color]: [color=yellow]" + globals.itemdict[essence].name + "[/color]"
 				###---End Expansion---###
 				globals.itemdict[essence].amount += 1
-			mana += i.orgasms*3 + rand_range(1,2)
+			#ralphC
+			#print('\nRalph Test: ' + str(i.orgasms) + ' orgasms had by: ' + str(i.person.name) + ' ') #ralphC temp
+			#print("Ralph Test: " + str(mana) + " mana before i.orgasms*3 added") #ralphC temp
+			mana += i.orgasms * variables.orgasmmana + rand_range(1,2) #ralphC - replaced 3 with variables.orgasmmana
+			#print("Ralph Test: " + str(mana) + " mana after i.orgasms*3 added (+1 or 2)") #ralphC temp
+			#print("Ralph Test: succubusdraincount = " + str(i.succubusdraincount) + ") for: " + str(i.person.name) + " ") #ralphC temp
+			mana -= i.succubusdraincount * variables.orgasmmana #ralphC - no mana from the orgasms eaten by Succubi
+			#print("Ralph Test: " + str(mana) + " mana after i.succubusdraincount*3 subtracted") #ralphC temp
+			i.succubusdraincount = 0
```

Comment:
There is no need to zero this value, it is never used again.

Reply:
comment incorporated for next commit